main/openldap/APKBUILD

```diff
@@ -9,7 +9,7 @@
 #   - CVE-2017-9287
 #
 pkgname=openldap
-pkgver=2.4.46
+pkgver=2.4.47
 pkgrel=2
```

Comment:
@bratkartoffel It's not a huge thing, but `pkgrel` should be set to `0` when upgrading.